packages/next/build/webpack/plugins/middleware-plugin.ts

```diff
@@ -352,18 +373,16 @@ function getCodeAnalyzer(params: {
         return
       }
 
-      if (dev) {
```

Comment:
question: does this change mean we'll enforce the "no eval" limitation in prod as well?

Reply:
Indeed, it means that we're also wrapping dynamic code in production as well.
While discussing this with @sokra, we realized that we could always instrument the code.
If it is reached in dev, it'll print the nice warning, telling users to either allow or remove it.

It could not be shipped in production previously (because the build was always failing). Now that users could allow it, having the nice warning in production is handy as well.

Reply:
Sounds nice. Thanks!